.travis.yml

```diff
@@ -46,12 +47,18 @@ matrix:
   include:
     - python: 3.6
       env: TOXENV=py36-core
+    - python: 3.6
+      env: TOXENV=py36-dropbox
     - python: 3.6
       env: TOXENV=py36-unixsocket
     - python: 3.7
       dist: xenial
       sudo: true
       env: TOXENV=py37-core
+    - python: 3.7
+      dist: xenial
+      sudo: true
+      env: TOXENV=py37-dropbox
```

Comment:
given inability to mock tests, could we remove dropbox from travis?

Reply:
I have not removed dropbox-related tests from travis.yml

However, I have modified Travis, so that it only runs the "_py??-dropbox_" builds when the `DROPBOX_APP_TOKEN` is defined. This means that dropbox tests will not even appear in the https://travis-ci.org/spotify/luigi/, but they will run in https://travis-ci.org/joseignaciorc/luigi/ (and in the Travis of any developer that wants to test or improve the DropboxTarget).

Reply:
thanks for this. good to know that functionality exists